README.md

```diff
@@ -13,6 +13,35 @@ Our goal is to document these sets of solutions to challenges, make it easy for
 These implementations will likely be documents, videos, or code, each of which is generic enough to be used across companies,
 yet specific enough that people in the company can leverage them to solve their problems without needing notable amounts of company-specific work.
 
+## Implementation
+
+We expect to create together **The Handbook of InnerSource**, which contains our collective, practical knowledge of how to run InnerSource projects and programs.
+The goal is that a new practioner can implement the instruction in the handbook and be successful with it in their InnerSource goals.
+We expect that the handbook will have various sections for the different personas and roles involved with InnerSource, e.g. ISPO group, engineering managers, product leaders, individual engineers, etc.
+The handbook may look something like the [Managing InnerSource Projects](https://innersourcecommons.gitbook.io/managing-inner-source-projects/) book and, in fact, may end up being a superset and subsume the content there.
```

Comment:
Maybe move this sentence to the "some comments on this approach" section.
It is a helpful side node for people that already know the other ISC resources but does not seem essential to understand the general idea of the approach suggested here.

Reply:
Good idea.